applications/OptimizationApplication/python_scripts/controls/thickness/shell_thickness_control.py

```diff
@@ -84,58 +36,48 @@ def __init__(self, name: str, model: Kratos.Model, parameters: Kratos.Parameters
 
         default_settings = Kratos.Parameters("""{
             "controlled_model_part_names": [],
-            "filter_settings": {
-                "type" : "implicit",
-                "radius": 0.000000000001,
-                "linear_solver_settings" : {}
-            },
-            "beta_value": 25.0,
-            "penalty_power": 1,
-            "output_all_fields": false,
-            "initial_physical_thickness":0.000001,
-            "physical_thicknesses": [],
-            "fixed_model_parts_and_thicknesses": {},
-            "utilities": []
+            "filter_settings"            : {},
```

Comment:
this PR is about the filters not changing the settings of the thickness control, please remove and put such changes into another separate PR. Again please do not change every thing without providing reasons and notices !

Reply:
beta settings have anything to do with update filters PR